doc/source/data/getting-started.rst

```diff
@@ -4,113 +4,144 @@ Getting Started
-
-These transformations are composable. You can further apply transformations on the
-output Dataset, forming a chain of transformations to express more complex logic.
-
-By default, transformations are executed using Ray tasks.
-For transformations that require setup, you may want to use Ray actors by specifying
-``compute=ray.data.ActorPoolStrategy(min, max)`` and Ray will use an autoscaling
-actor pool of ``min`` to ``max`` actors to execute your transforms. This will cache
-the stateful setup at the actor creation time, which is particularly useful if the
-setup is expensive.
-
-See the :ref:`Transforming Datasets guide <transforming_datasets>` for an in-depth guide
-on transforming datasets.
-
-Accessing and exchanging datasets
----------------------------------
-
-Datasets can be passed to Ray tasks or actors and accessed with
-:meth:`~ray.data.Dataset.iter_batches` or
-:meth:`~ray.data.Dataset.iter_rows`.
-This does not incur a copy, since the blocks of the Dataset are passed by reference
-as Ray objects:
-
-.. literalinclude:: ./doc_code/quick_start.py
-   :language: python
-   :start-after: __data_access_begin__
-   :end-before: __data_access_end__
-
-Datasets can be split up into disjoint sub-datasets.
-Locality-aware splitting is supported if you pass in a list of actor handles to the
-:meth:`~ray.data.Dataset.split` function along with the number of desired
-splits.
-This is a common pattern useful for loading and splitting data between distributed
-training actors:
-
-.. literalinclude:: ./doc_code/quick_start.py
-   :language: python
-   :start-after: __dataset_split_begin__
-   :end-before: __dataset_split_end__
-
-See the :ref:`Consuming Datasets guide <consuming_datasets>` for an in-depth guide
-on accessing and exchanging datasets.
+Apply :ref:`user-defined functions <transform_datasets_writing_udfs>` (UDFs) to
+transform datasets. Ray executes transformations in parallel for performance at scale.
+
+.. testcode::
+
+    import pandas as pd
+
+    # Find rows with spepal length < 5.5 and petal length > 3.5.
+    def transform_batch(df: pd.DataFrame) -> pd.DataFrame:
+        return df[(df["sepal length (cm)"] < 5.5) & (df["petal length (cm)"] > 3.5)]
+
+    transformed_dataset = dataset.map_batches(transform_batch)
+    print(transformed_dataset)
+
+.. testoutput::
+
+    MapBatches
+    +- Dataset(num_blocks=1, num_rows=150, schema={sepal length (cm): double, sepal width (cm): double, petal length (cm): double, petal width (cm): double, target: int64})
+
+To learn more about transforming datasets, read
+:ref:`Transforming Datasets <transforming_datasets>`.
+
+Consume the dataset
+-------------------
+
+Pass datasets to Ray tasks or actors, and access records with methods like
+:meth:`~ray.data.Dataset.iter_batches`.
+
+.. tabbed:: Local
+
+    .. testcode::
+
+        batches = transformed_dataset.iter_batches(batch_size=8)
+        print(next(iter(batches)))
```

Comment:
Somehow the rendered page has a weird extra gray line at the left. This is only happening for `Local` tab (but not `Tasks` / `Actors` tabs). Do you know what causes this?

<img width="954" alt="Screen Shot 2023-01-30 at 4 18 20 PM" src="https://user-images.githubusercontent.com/4629931/215626396-b5f41549-b430-4a4d-8371-ca7cfbe706c0.png">